doc/xml/daps_user_create.xml

```diff
@@ -46,7 +46,7 @@
  <para>
   By default, the initialization script creates a <sgmltag>book</sgmltag> as
   example document. With the <option>-r</option> option, you can also set
```

Comment:
I think, the `-r` option is not available, only `--rootelement`. Either you add this option to the script or you change it in the doc. :wink: